docs/docs/user-setup.md

```diff
@@ -143,6 +143,10 @@ GitHub Repo: https://github.com/petejohanson/zmk-config.git
 
 Only the GitHub username is required; if you are happy with the defaults offered in the square brackets, you can simply hit `Enter`.
 
+:::note
+If you are using SSH keys for git push, change GitHub Repo field to the SSH scheme, e.g. `git@github.com:petejohanson/zmk-config.git'.
```

Comment:
One typo I had failed to notice earlier:
```suggestion
If you are using SSH keys for git push, change GitHub Repo field to the SSH scheme, e.g. `git@github.com:petejohanson/zmk-config.git`.
```
You can see how it renders here: https://deploy-preview-1089--zmk.netlify.app/docs/user-setup/#github-details